tests/unnamed_struct_serialize.rs

```diff
@@ -14,8 +15,7 @@ fn main() {
     let person = Person("joana".to_string(), 290000, Kind::Chill);
 
     assert_eq!(
-        edn_rs::to_string(person),
-        ":person{ 0 \"joana\", 1 290000, 2 :kind/chill, }"
+        person.serialize(),
```

Comment:
The idea of `edn::from_str` & `edn::to_string` is that we follow a similar API to what serde does, you never call `serialize`/`deserialize` directly.

Reply:
I don't think it makes sense to have this for `edn-rs` though? I believe it's that way because something like serde-json requires `serde` https://docs.rs/serde/1.0.193/serde/ser/trait.Serialize.html. You're supposed to be able to use `serde-json` without having to construct or use dependencies from `serde`. We don't have this, and everything is defined in one crate.

We also have a similar thing with deserialize where it's easier to
`let person = Person::deserialize(edn);` than
`let person: Person = edn_rs::from_str(edn);`